### ISSUE TITLE
table-100's have a lot of padding

### ISSUE THREAD

**Issue by jloh** [kepano/obsidian-minimal#586]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

7.0.3

### Describe the bug

I have DataviewJS tables that are grouped by and displayed under headings. They have long rows and I use `wide` and `table-100` classes along with `table-overflow-x` which just sets `overflow-x: auto` so that all the separate tables scroll separately without the whole page scrolling.

With the latest update Minimal sets a `min-width` on tables that cause a lot of dead space.

If I make my viewport small (like on mobile for example) it makes a large part of the table blank. I can fix this by setting `min-width` to `auto` like this:

```css
.markdown-preview-view.is-readable-line-width .markdown-preview-sizer table,
.markdown-source-view.mod-cm6.is-readable-line-width
	.cm-contentContainer.cm-contentContainer
	> .cm-content
	table {
	min-width: auto;
}
```

Just reporting as I'm not sure if this is intentional. Understand if it is.

Current state:

<img width="528" alt="image" src="https://github.com/kepano/obsidian-minimal/assets/3689532/35f79777-6f81-4f21-bea4-8887fa4e2070">

After CSS:

<img width="506" alt="image" src="https://github.com/kepano/obsidian-minimal/assets/3689532/64f0ecda-b48c-44fa-9936-e83f6a3bba6b">

### Debug info

```
SYSTEM INFO:
	Obsidian version: v1.3.7
	Installer version: v1.3.5
	Operating system: Darwin Kernel Version 22.5.0: Mon Apr 24 20:52:24 PDT 2023; root:xnu-8796.121.2~5/RELEASE_ARM64_T6000 22.5.0
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal v7.0.3
	Snippets enabled: 6
	Restricted mode: off
	Plugins installed: 26
	Plugins enabled: 18
		1: Dataview v0.5.56
		2: QuickAdd v1.2.1
		3: Minimal Theme Settings v6.3.1
		4: Style Settings v1.0.6
		5: Default New Tab Page v0.11.0
		6: Templater v1.16.0
		7: Calendar v1.5.10
		8: Force note view mode v1.1.2
		9: Periodic Notes v0.0.17
		10: Banners v1.3.3
		11: Folder notes v1.3.4
		12: MetaEdit v1.8.0
		13: Plugin Manager v0.1.5
		14: Book Search v0.5.10
		15: Sortable v0.3.1
		16: Tasks v4.0.1
		17: Cron v1.1.1
		18: Linter v1.16.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by jloh** [comment]
Just want to say that I'm a big fan of version 7! Previously with this kind of config (multiple tables on the one page) it could end up with some being smaller than previous ones if the rows weren't long enough and it looked weird. Now they're all at least 100% of the view width which makes it look fantastic. Thanks so much of Minimal ❤️

**Comment by kepano** [comment]
Try 7.0.5 and let me know if it's improved

**Comment by jloh** [comment]
Thankyou! Can confirm its fixed in 7.0.5
